cov.el

```diff
@@ -173,32 +167,39 @@ If `cov-coverage-file' is non nil, the value of that variable is returned. Other
   (or cov-coverage-file
       (setq cov-coverage-file (cov--locate-coverage (f-this-file)))))
 
-(defun cov--keep-line? (line)
-  (s-matches? cov-line-re line))
-
 (defun cov--read (file-path)
-  "Read a gcov file, filter unused lines, and return a list of lines"
-  (cl-remove-if-not
-   'cov--keep-line?
-   (cov--read-lines file-path)))
-
-(defun cov--parse (string)
-  "Returns a list of (line-num, times-ran)"
-  `(,(string-to-number (nth 2 (s-match cov-line-re string)))
-    ,(string-to-number (nth 1 (s-match cov-line-re string)))))
-
-(defun cov-make-overlay (line fringe help)
+  "Read a gcov file as buffer and return that buffer."
+  (find-file-noselect file-path))
```

Comment:
``` elisp
(with-temp-buffer (insert-file-contents file-path) ...) 
```
in gcov--parse should improve performance. It might be smart to merge these functions.